code/core/arm9/source/GbaColorCorrection/ColorProfiles.h

```diff
@@ -0,0 +1,159 @@
+#pragma once
+#include "Core/Math/fixed.h"
+
+struct ColorProfile
+{
+    fix32<12> matrix[3][3]; // 3x3 correction matrix
+    fix32<12> luminance;    // Luminance factor
+};
+
+// Libretro shader values. Credits: hunterk and Pokefan531.
+// https://forums.libretro.com/t/real-gba-and-ds-phat-colors/1540/238
+// Last updated 10-05-2025.
+
+// const ColorProfile PRESET_NAME = 
+//{
+//    {
+//        { [r], [gr], [br] },
+//        { [rg], [g], [bg] },
+//        { [rb], [gb], [b] }
+//    },
+//    luminance
+//};
+
+// libretro GBA AGB-001 color (sRGB).
+const ColorProfile Agb001 =
```

Comment:
Move the color profiles to a cpp file. Data should not be in header files. In this header file only put the forward declarations such that they can be referenced elsewhere.